kinto/plugins/history/views.py

```diff
@@ -57,3 +61,85 @@ def _extract_filters(self):
             filters_str_id.append(filt)
 
         return filters_str_id
+
+
+version_view = Service(name='version_view',
+                       description='Handle retrieving object from the past.',
+                       path='{subpath:.*}/version/{last_modified:[0-9]{13}}')
+
+
+@version_view.get(permission=NO_PERMISSION_REQUIRED)
```

Comment:
> The history endpoint already handles this, by providing history entries only on data that was readable/writable at the time it was edited.

If this is the case it is really great ! We can have the exact same behaviour here.